src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -697,18 +697,16 @@ public static string NormalizePathPrefix(string filePath, ImmutableArray<KeyValu
         /// 
         /// The more accurate way is to let the framework parse the path and throw on any errors.
         /// </summary>
-        /// <param name="path"></param>
-        /// <returns></returns>
-        public static bool IsValidFilePath(string path)
+        public static bool IsValidFilePath(string fullPath)
```

Comment:
>public static bool IsValidFilePath(string fullPath) [](start = 8, length = 51)

Would it be valid to add an assert ```Debug.Assert(IsAbsolute(fullPath));```?

Reply:
Yes. Will add that.

---
In reply to: [192184509](https://github.com/dotnet/roslyn/pull/27279#discussion_r192184509) [](ancestors = 192184509)